### ISSUE TITLE
About learning rate schedule of training

### ISSUE THREAD

**Issue by yeliudev** [GothicAi/Instaboost#19]
Hi @Fang-Haoshu ! Thanks for your great work on Instaboost!

I would like to train a Mask-RCNN with Instaboost and some other tricks on my own using mmdetection, but I'm considering whether it is necessary to train the model for 4x, which is maybe too costly. May I ask that have you tried training the models for fewer epochs (2x or 3x)? If so, could you please report the results? Thank you!

**Comment by Fang-Haoshu** [comment]
Hi, thanks. Yes, we have tried. It can also boost the results. The results are listed in the ablation study in our paper. You can refer to the last page in our paper.

**Comment by yeliudev** [comment]
I see, thank you so much!
